user_management/api/serializers.py

```diff
@@ -31,6 +38,7 @@ class Meta:
 
 
 class RegistrationSerializer(ValidateEmailMixin, serializers.ModelSerializer):
+    email = serializers.EmailField(validators=[unique_email_validator])
```

Comment:
maybe the `email` field can be defined in the mixin? (Same apply to `UserSerializerCreate`)